src/components/global/Playground/stackblitz.utils.ts

```diff
@@ -92,18 +108,40 @@ const openAngularEditor = async (code: string, options?: EditorOptions) => {
     'angular/angular.json',
     'angular/tsconfig.json',
     'angular/package.json'
-  ], options.version)
+  ], options.version);
+
+  const appModule = 'src/app/app.module.ts';
+  const files = {
+    'src/main.ts': defaultFiles[0],
+    'src/polyfills.ts': `import 'zone.js/dist/zone';`,
+    [appModule]: defaultFiles[1],
+    'src/app/app.component.ts': defaultFiles[2],
+    'src/app/app.component.css': defaultFiles[3],
+    'src/app/app.component.html': defaultFiles[4],
+    'src/app/example.component.ts': defaultFiles[5],
+    'src/app/example.component.html': code,
+    'src/app/example.component.css': '',
+    'src/index.html': '<app-root></app-root>',
+    'src/styles.css': defaultFiles[6],
+    'src/global.css': defaultFiles[7],
+    'src/theme/variables.css': defaultFiles[8],
+    'angular.json': defaultFiles[9],
+    'tsconfig.json': defaultFiles[10],
+    ...options?.files
+  };
+
+  const package_json = defaultFiles[11];
 
   if (options.angularModuleOptions) {
     if (options.angularModuleOptions.imports) {
-      app_module_ts = `${options.angularModuleOptions.imports.join('\n')}\n${app_module_ts}`;
+      files[appModule] = `${options.angularModuleOptions.imports.join('\n')}\n${files[appModule]}`;
     }
     if (options.angularModuleOptions.declarations) {
-      app_module_ts = app_module_ts.replace('/* CUSTOM_DECLARATIONS */', options.angularModuleOptions.declarations.map(d => `\n  ${d}`).join(','));
+      files[appModule] = files[appModule].replace('/* CUSTOM_DECLARATIONS */', options.angularModuleOptions.declarations.map(d => `\n  ${d}`).join(','));
```

Comment:
I'm not really sure what this replace is for; the `/* CUSTOM_DECLARATIONS */` piece wasn't present in the template even before my changes. I left it as-is just in case. Functionality should be the same as before.

Reply:
This is a legacy feature that we never ended up needing. Before the multi-file playgrounds were a thing, I had experimented with a configuration object that would just append the import statements and declarations to your existing playground code, instead of having to re-create the `app.module.ts` yourself. 

I think we could kill this feature (it isn't used in any playground to date).

Reply:
Does that include the whole `angularModuleOptions` feature? Or just this `CUSTOM_DECLARATIONS` piece? If the former, I'll whip up a separate PR for it later 👀

Reply:
Yes, it would be the entire `angularModuleOptions` configuration/feature.